frontend/scss/style.scss

```diff
@@ -1,7 +1,26 @@
+$silver : #A6DED9 !default;
+$gold : #DED9A6 !default;
+
+$highlight: #8b9ae0 !default;
+$selection: #8b9ae0 !default;
+
+$dark-square : #000 !default;
+$light-square : #fff !default;
+$selected-dark-square : #009 !default;
+$selected-light-square : #00f !default;
+$light-trap : #f00 !default;
+$dark-trap : #f00 !default;
+
+//$board-bg : "../images/boards/grey.jpg" !default;
+//$board-bg : "../images/boards/BoardStoneBig.jpg" !default;
+$board-bg : "../images/boards/boardmarbletiled.png" !default;
+
+$board-size : 480px !default;
+$step-to-img : "../images/green-circle.png" !default;
```

Comment:
I deleted some of the values. It might make sense for silver and gold to be in here since we may need them for other parts.